src/index.js

```diff
@@ -61,11 +64,20 @@ const formatOptions = ({steps, ...otherOptions}) => ({
   steps: (steps || ['welcome','document','face','complete']).map(formatStep)
 })
 
+const setI18n = (options) => {
+  const locale = options.language ? options.language.locale : 'en'
+  const customStrings = options.language ? options.language.customStrings : null
+  const phrases = locales[locale]
+  const polyglot = new Polyglot({locale, phrases})
```

Comment:
why do we need to pass the locale in?

Reply:
That's only required to handle pluralisation, but we aren't pluralising any string at the moment. I can remove it for now and re-introduce it if we need it for Spanish translations.